include/aspect/initial_temperature/lithosphere_mask.h

```diff
@@ -33,6 +33,84 @@ namespace aspect
   {
     using namespace dealii;
 
+    namespace LABDepth
+    {
+      template <int dim>
+      class LABDepthLookup : public SimulatorAccess<dim>
+      {
+        public:
+          /**
+           * Empty Constructor.
+           */
+          LABDepthLookup ();
+
+          /**
+           * Initialization function. This function is called once at the
+           * beginning of the program. Checks preconditions.
+           */
+          void
+          initialize ();
+
+          /**
+           * Return LAB depth as a function of position (latitude and longitude). For the
+           * current class, this function returns value from the text file. We read in
```

Comment:
Please update the comment to say it either returns the value from the file, or a constant value depending on the input parameters.